web/src/invite.ts

```diff
@@ -15,13 +15,19 @@ import {csrf_token} from "./csrf";
 import * as dialog_widget from "./dialog_widget";
 import * as gear_menu from "./gear_menu";
 import {$t, $t_html} from "./i18n";
+import * as overlays from "./overlays";
 import {page_params} from "./page_params";
+import * as pill_typeahead from "./pill_typeahead";
```

Comment:
I am not sure about the best way to resolve circular dependency here.

Reply:
I think the best option likely involves converting a bunch more modules to TypeScript, such that we can do this import. But the alternative plan is to turn `invite.ts` back into JS. Discussed in https://chat.zulip.org/#narrow/stream/6-frontend/topic/input_pill.20typescript/near/1670357